deployments/helm/nvidia-device-plugin/templates/daemonset-gfd.yml

```diff
@@ -182,6 +186,10 @@ spec:
             mountPath: "/etc/kubernetes/node-feature-discovery/features.d"
           - name: host-sys
             mountPath: "/sys"
+        {{- if .Values.imex.enabled }}
+          - name: imex-nodes-config
+            mountPath: {{ .Values.imex.configFile | quote }}
+        {{- end }}
```

Comment:
I'm not sure if we need an `enabled` flag around any of these mounting directives. Instead we should wrap it with a `{{- if typeIs "string" .Values.imexNodesConfigFile }}` such that when this is `nil` (the default), we use the default value for that is set for this file in the CLI.

Reply:
Noted, will edit

Reply:
Actually thinking about this again -- we alway need to mount (at least) the folder for this, don't we? Otherwise the file won't be made available in the container (even though there is a default set path for it in the binarx).